src/main/java/org/terracotta/tinypounder/TinyPounderMainUI.java

```diff
@@ -1192,25 +1194,25 @@ private void generatePropertiesFile(boolean skipConfirmOverwrite) {
           sb.append(nodePrefix + "hostname=" + hostName.getValue() + "\n");
           sb.append(nodePrefix + "port=" + clientPort.getValue() + "\n");
           sb.append(nodePrefix + "group-port=" + groupPort.getValue() + "\n");
-          sb.append(nodePrefix + "log-dir=" + logs.getValue() + "\n");
+          sb.append(nodePrefix + "log-dir=" + logs.getValue().replace("\\", "\\\\") + System.lineSeparator());
 
           // For testing
           sb.append(nodePrefix + "tc-properties=logging.maxBackups:10,logging.maxLogFileSize:256\n");
           sb.append(nodePrefix + "logger-overrides=org.terracotta.management:INFO,com.terracottatech.management:INFO\n");
 
           if (platformBackup.getValue()) {
             TextField path = (TextField) dataRootGrid.getComponent(DATAROOT_PATH_COLUMN, getBackupRow());
-            sb.append(nodePrefix + "backup-dir=" + path.getValue() + "\n");
+            sb.append(nodePrefix + "backup-dir=" + path.getValue().replace("\\", "\\\\") + System.lineSeparator());
```

Comment:
You can make `System.lineSeparator` a static import, and use it everywhere in place of "\n".